packages/ChangesReloaded-Core.package/CRChangeRecordLoader.class/instance/loadFrom.to.onProgress..st

```diff
@@ -0,0 +1,16 @@
+loading
+loadFrom: startPosition to: stopPosition onProgress: aBlock
+	
+	| prevChar |
+	self clearChangeRecords.
+	file position: startPosition.
+	[file position < stopPosition] whileTrue: [
```

Comment:
Should those be round brackets?

Reply:
Nope, because the block should not only be evaluated to `true` or `false` and the `whileTrue:` should be called in that value.
Instead, the block should be evaluated after every iteration and the result of it should be used.
That's why `whileTrue:` is defined on `BlockClosure` rather than `Boolean`